novawallet/Modules/Vote/Governance/Validating/GovernanceValidatorFactory.swift

```diff
@@ -300,6 +307,34 @@ extension GovernanceValidatorFactory: GovernanceValidatorFactoryProtocol {
             return !tracks.isEmpty && tracks.isSubset(of: delegatingTracks)
         })
     }
+
+    func voteMatchesConviction(
+        with newVote: ReferendumNewVote?,
+        selectedConviction: ConvictionVoting.Conviction?,
+        convictionUpdateClosure: @escaping () -> Void,
+        locale: Locale?
+    ) -> DataValidating {
+        ErrorConditionViolation(onError: { [weak self] in
```

Comment:
Now, I got it why we are rerunning the validation. The validation must be WarningConditionalViolation and a user can continue not aborting the runner

Reply:
Oh, now I see. Yes, makes sense.